cli/command/service/formatter_test.go

```diff
@@ -33,29 +33,37 @@ func TestServiceContextWrite(t *testing.T) {
 		// Table format
 		{
 			formatter.Context{Format: NewListFormat("table", false)},
-			`ID                  NAME                MODE                REPLICAS            IMAGE               PORTS
-id_baz              baz                 global              2/4                                     *:80->8080/tcp
-id_bar              bar                 replicated          2/4                                     *:80->8080/tcp
+			`ID                  NAME                MODE                REPLICAS               IMAGE               PORTS
+02_bar              bar                 replicated          2/4                                        *:80->8090/udp
+01_baz              baz                 global              1/3                                        *:80->8080/tcp
+04_qux2             qux2                replicated          3/3 (max 2 per node)                       
+03_qux10            qux10               replicated          2/3 (max 1 per node)                       
 `,
 		},
 		{
 			formatter.Context{Format: NewListFormat("table", true)},
-			`id_baz
-id_bar
+			`02_bar
+01_baz
+04_qux2
+03_qux10
 `,
 		},
 		{
-			formatter.Context{Format: NewListFormat("table {{.Name}}", false)},
-			`NAME
-baz
-bar
+			formatter.Context{Format: NewListFormat("table {{.Name}}\t{{.Mode}}", false)},
+			`NAME                MODE
+bar                 replicated
+baz                 global
+qux2                replicated
+qux10               replicated
```

Comment:
Sorting moved to the formatter (see the PR description), which is why these tests now have their output in a different order. I added services named `qux2` and `qux10` to verify the formatter uses naturalsort (otherwise `qux2` would come after `qux10`)